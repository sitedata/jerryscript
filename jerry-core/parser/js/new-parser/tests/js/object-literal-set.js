/* This test contains object literal parser tests. */

var a = { set set() {} }
