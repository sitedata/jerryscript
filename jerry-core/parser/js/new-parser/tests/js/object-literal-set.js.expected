
--- Script parsing start ---

  [  1] CBC_CREATE_OBJECT

--- Function parsing start ---


--- Script parsing end ---

Parse error 'Property setters must have one argument.' at line: 3 col: 21
